src/main/java/randoop/generation/OrienteeringSelection.java

```diff
@@ -0,0 +1,157 @@
+package randoop.generation;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import randoop.sequence.ExecutableSequence;
+import randoop.sequence.Sequence;
+import randoop.util.Randomness;
+import randoop.util.SimpleList;
+
+/**
+ * Implements the Orienteering component, as described by the paper "GRT: Program-Analysis-Guided
+ * Random Testing" by Ma et. al (appears in ASE 2015):
+ * https://people.kth.se/~artho/papers/lei-ase2015.pdf .
+ *
+ * <p>Biases input selection towards sequences that have lower execution cost. Execution cost is
+ * measured by the number of method calls in a sequence and the time it takes to execute.
```

Comment:
Corrected implementation to explicitly compute the execution time of each input sequence rather than relying on the execution time of the complete sequence.